.travis.yml

```diff
@@ -12,38 +12,28 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DEPS=lowest
-    - php: 5.6
-      env:
-        - DEPS=latest
-    - php: 7
-      env:
-        - DEPS=lowest
-    - php: 7
-      env:
-        - DEPS=latest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7.1
-      env:
-        - DEPS=latest
         - CS_CHECK=true
         - TEST_COVERAGE=true
-    - php: 7.2
+    - php: 7.3
+      env:
+        - DEPS=latest
+    - php: 7.4
       env:
         - DEPS=lowest
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.3
+    - php: 8.0
       env:
         - DEPS=lowest
-    - php: 7.3
+        - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
laminas-servicemanager does not install without `--ignore-platform-reqs` I can remove or remove when SM adds support for php8?

Reply:
[SMv3.6 has been released](https://github.com/laminas/laminas-servicemanager/releases/tag/3.6.0) with PHP8 support, can you drop `--ignore-platform-reqs` and see if CI is now happy?

Reply:
@Slamdunk updated but for some reason, tests did not start

Reply:
There's only one reason: Travis. Let's wait 24 hours and see...

Reply:
@Slamdunk 
Quota is exhausted, for the rest of the month.

> Since then, Travis-CI has announced that OSS plans are limited to 10k minutes per month, and we have twice reached the limit by mid-month.

See: https://github.com/laminas/technical-steering-committee/issues/61